src/components/molecules/Visualizer/Widget/Storytelling/index.tsx

```diff
@@ -160,7 +179,8 @@ const Current = styled(Flex)`
   }
 `;
 
-const Title = styled(Text)`
+const Title = styled(Text)<{ color: string }>`
+  color: ${({ color }) => color}!important;
```

Comment:
The `!important` isn't necessary. Above where you use Title just pass `customColor` and you should be good to go!
ie
```
<Title color={publishedTheme.mainText} size="m" weight="bold" customColor>
....
```